agent/proxycfg/state.go

```diff
@@ -38,6 +38,7 @@ const (
 	coalesceTimeout                    = 200 * time.Millisecond
 	rootsWatchID                       = "roots"
 	leafWatchID                        = "leaf"
+	peerTrustBundleWatchID             = "peer-trust-bundle:"
```

Comment:
Since this is a prefix the other prefix style names would suggest including "prefix" in the var name somewhere like `peerTrustBundleIDPrefix`